src/node.cc

```diff
@@ -704,6 +704,12 @@ const char *signo_string(int signo) {
 # endif
 #endif
 
+#ifdef SIGINFO
+# if SIGINFO != SIGPWR
```

Comment:
That defense is not necessary — the C/C++ preprocessors evaluate all undefined symbols to `0`. See [here](http://stackoverflow.com/a/5085425).

It's worth noting that the code immediately preceding this ([Lines 689-705](https://github.com/nodejs/node/pull/6093/files/afc3a44c7e26f3af60af0c1a976843c26ce82e05#diff-cd53544f44aab2c697bcd7b6a57f23ccL689)) employs the same bare `#if` where the RHS may be undefined. Specific example: SIGPWR is defined on Linux, but not SIGLOST ([Source](http://www.skrenta.com/rt/man/signal.7.html)).